src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -22,7 +22,7 @@ public class DeleteCommand extends Command {
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_DELETE_RECIPE_SUCCESS = "Deleted Recipe: ";
+    public static final String MESSAGE_DELETE_RECIPE_SUCCESS = "Deleted Recipe: %1$s";
 
     private final Index targetIndex;
```

Comment:
I feel that it may not be a good idea to represent the uuid as an index, since Index class contains methods that are perhaps unnecessary since uuid will always be one based.